app/vmalert/README.md

```diff
@@ -95,6 +95,11 @@ name: <string>
 extra_filter_labels:
   [ <labelname>: <labelvalue> ... ]
 
+# Optional list of labels added to every rule within a group.
+# It has priority over external.label flag values and rule labels.
```

Comment:
Not sure if priorities are correct. WDYT about 
`external.label` > `group.labels` > `rule.labels`, where rule has the lowest priority, group has more priority than rule, and external.label overrides any of them?

Reply:
I think the priority should be inverse, e.g. per-rule labels should override per-group labels, while per-group labels should override `external.label`. In this case it will be possible customizing labels per each group and per each rule. Otherwise the per-rule labels customization is overridden by per-group labels and `external.label`, which looks like not what most users want.

Reply:
Agree, thanks!